dashboard/src/components/Table/TableStatusFilter.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import { useMemo } from 'react';
+
+import { Button } from '@/components/ui/button';
+
+interface ITableStatusFilter {
+  filters: {
+    onClick: () => void;
+    label: string;
+    isSelected: boolean;
+  }[];
+}
```

Comment:
I thing would be better receive just one OnClick function that receives the filter value
```suggestion
interface ITableStatusFilter {
  onClick: (value: string) => void;
  filters: {
    label: string;
    value?: string;
  }[];
}
```
and then in the component we can do `onClick(filter.value || filter.label)`